pennylane/measurements/mid_measure.py

```diff
@@ -82,6 +82,29 @@ def func():
         Python ``not``, ``and``, ``or``, do not work since these do not have dunder methods.
         Instead use ``~``, ``&``, ``|``.
 
+    Mid-circuit measurement results can be processed with the usual measurement functions such as
+    :func:`~.expval`. For QNodes with finite shots, :func:`~.sample` applied to a mid-circuit measurement
+    result will return a binary sequence of samples.
+    See 
+    `here <https://xanaduai-pennylane--5441.com.readthedocs.build/en/5441/introduction/measurements.html#mid-circuit-measurement-statistics>`_  
```

Comment:
Maybe make this a dynamic cross reference?

Reply:
Ah! Sorry hahaha, this was the link Tom referenced above, but that's the PR preview lol. I'll fix!